src/assets/styles/transversals/_footer.scss

```
@@ -51,6 +51,114 @@
   }
 }
 
+@keyframes ghostEffect {
+  0% {
+    transform: translateY(0);
+    //filter: blur(5px);
+    opacity: 1;
+  }
+  50% {
+    transform: translateY(-10px);
+    //filter: blur(2px);
+    opacity: 1;
+  }
+  100% {
+    transform: translateY(-100);
+    //filter: blur(0px);
+    opacity: 1;
+  }
+}
+
+@keyframes slideUp {
+  from {
+    transform: translateY(100%);
+    opacity: 0;
+  }
+  to {
+    transform: translateY(0);
+    opacity: 1;
+  }
+}
+
+@keyframes blurEffect {
+  from {
+    filter: blur(50px);
+  }
+  to {
+    filter: blur(0px);
+  }
+}
+
+
+.chatBotOracle {
+  position: fixed;
+  z-index: 99;
+  bottom: 120px;
+  right: 30px;
+}
+
+.chatBotImage {
+  cursor: pointer;
+  transition: all 0.5s ease;
+  border: solid 5px white;
+  opacity: 0.5;
+  border-radius: 100px;
+  position: relative; /* Ajout pour le positionnement du texte */
+
+  &:hover{
+    opacity: 1;
+  }
+
+  &:hover .chatBotText {
+    visibility: visible;
+    opacity: 1;
+    animation: slideUp 0.5s ease forwards;
+  }
+    
+  img{
+    border-radius: 100px;
+  }
+}
+
+.chatBotText {
+  visibility: hidden;
+  font-family: 'Norse';
+  font-size: 25px;
+  width: 120px;
+  color: white;
+  text-align: center;
+  border-radius: 5px;
+  padding: 5px 0;
+  position: absolute;
+  z-index: 99;
+  bottom: 150%;
+  left: 50%;
+  margin-left: -60px;
+  opacity: 0;
+  display: inline-block;
+  transform: translateY(20px); /* Début de l'effet de translation */
+  opacity: 0;
+  visibility: hidden;
+  opacity: 0;
+  position: absolute;
+  bottom: 100%; /* Départ de l'animation depuis le bas de l'image */
+  left: 50%;
+  transform: translateX(-80%);
+  transition: visibility 0s, opacity 0.5s linear;
+  animation:blurEffect 2s ease-in-out;
+}
```

Remove duplicated properties.

```suggestion
.chatBotText {
  font-family: 'Norse';
  font-size: 25px;
  width: 120px;
  color: white;
  text-align: center;
  border-radius: 5px;
  padding: 5px 0;
  z-index: 99;
  margin-left: -60px;
  display: inline-block;
  visibility: hidden;
  opacity: 0;
  position: absolute;
  bottom: 100%;
  left: 50%;
  transform: translateX(-80%);
  transition: visibility 0s, opacity 0.5s linear;
}
```

---

Ok thanks ;)